### PR TITLE
bpo-29886: Link bytes.{,to}hex from binascii docs

### PR DESCRIPTION
**Pull request by chrysn**
A suggested fix for http://bugs.python.org/issue29886 which I reported earlier today

https://bugs.python.org/issue29886

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](http://bugs.python.org/) (b.p.o) to verify you have signed the CLA. This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://cpython-devguide.readthedocs.io/pullrequest.html#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

File: Doc/library/binascii.rst [hunk 1]
```diff
@@ -158,6 +160,9 @@ The :mod:`binascii` module defines the following functions:
    of hexadecimal digits (which can be upper or lower case), otherwise an
    :exc:`Error` exception is raised.
 
+   Similar functionality (accepting only strings arguments, but more liberal
```
Comment (vadmium) [inline Doc/library/binascii.rst]: string arguments [not strings]

Or even “text string arguments” to be clearer (but other people seem happy assuming “string” excludes byte strings, so maybe that does not matter).

**Comment by chrysn** [comment]
Updated accordingly. I originally had "text string" there and went for "string" because it is more prevalent in the rest of the documentation, but here the disambiguation could help.

@the-knights-who-say-ni, please review CLA state, it's signed and uploaded. (Maybe it helps to ping the bot).

**Review by rhettinger — APPROVED** [review]

**Comment by rhettinger** [comment]
For some reason, we can't get this past the CI checks.  Reapplying in a separate PR, https://github.com/python/cpython/pull/11319

Thanks for the work.

**Pull request closed without merging**